pkg/kdump/kdump-view.jsx

```diff
@@ -391,21 +346,21 @@ export class KdumpPage extends React.Component {
             if (target.multipleTargets) {
                 kdumpLocation = _("invalid: multiple targets defined");
             } else {
-                if (target.target == "local") {
+                if (target.type == "local") {
                     if (target.path)
                         kdumpLocation = cockpit.format(_("locally in $0"), target.path);
                     else
                         kdumpLocation = cockpit.format(_("locally in $0"), "/var/crash");
                     targetCanChange = true;
-                } else if (target.target == "ssh") {
+                } else if (target.type == "ssh") {
                     kdumpLocation = _("Remote over SSH");
                     targetCanChange = true;
-                } else if (target.target == "nfs") {
+                } else if (target.type == "nfs") {
                     kdumpLocation = _("Remote over NFS");
                     targetCanChange = true;
-                } else if (target.target == "raw") {
+                } else if (target.type == "raw") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17544-20220712-132416-8611a291-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)